src/script/components/ParticipantAvatar.tsx

```diff
@@ -68,22 +68,23 @@ export const INITIALS_SIZE = {
   [AVATAR_SIZE.XXX_SMALL]: '8px',
 };
 
-export interface ParticipantAvatarProps {
+export interface ParticipantAvatarProps extends React.ComponentProps<'div'> {
```

Comment:
```suggestion
export interface ParticipantAvatarProps extends React.HTMLProps<HTMLDivElement> {
```

Reply:
Ok, shall I convert all other ` React.ComponentProps<'div'>` from the webapp to `React.HTMLProps<HTMLDivElement>` as well?